e2e/tests/e2e/local_node.rs

```diff
@@ -77,24 +77,24 @@ impl<T: Transport> Resetter<T> {
 }
 
 #[derive(Debug, Clone)]
-pub struct GanacheApi<T> {
+pub struct LocalNodeApi<T> {
```

Comment:
uber-naming-nit: Maybe call this `EvmApi` because of the `evm_` prefix to the RPC calls to be consistent with the other RPC namespaces (like `Eth`, `Net`, etc.).